server/chat/background/task.py

```
@@ -1120,6 +1121,34 @@ def _update_session_status(session_id: str, status: str) -> None:
+            with conn.cursor() as cursor:
+                cursor.execute(
+                    """UPDATE incident_suggestions
+                       SET execution_status = %s
+                       WHERE execution_session_id = %s::uuid
+                         AND execution_status IN ('in_progress', 'executed')""",
+                    (status, session_id),
+                )
+                if cursor.rowcount > 0:
+                    conn.commit()
+                    logger.info(f"[BackgroundChat] Updated suggestion execution_status to '{status}' for session {session_id}")
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)